### PR TITLE
[Doc enhancement][Mobile and desktop tab switch][1843672]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Updated files. Please review.

### CODE REVIEW

File: msteams-platform/concepts/device-capabilities/native-device-permissions.md [hunk 1]
```diff
@@ -168,14 +168,14 @@ For example:
...
-
-   ![Tabs desktop device permissions prompt](~/assets/images/tabs/device-permissions-prompt.png)
-
 # [Mobile](#tab/mobile)
 
    ![Tabs mobile device permissions prompt](../../assets/images/tabs/MobileLocationPermission.png)
 
+# [Desktop](#tab/desktop)
+
+   ![Tabs desktop device permissions prompt](~/assets/images/tabs/device-permissions-prompt.png)
```
Comment (shuchauh) [inline msteams-platform/concepts/device-capabilities/native-device-permissions.md:177]: Would be great if we can update this image

Comment (Rajeshwari-v) [reply msteams-platform/concepts/device-capabilities/native-device-permissions.md:177]: @shuchauh, updated the mobile specific images.

**Review by heath-hamilton — COMMENTED** [review]
@Rajeshwari-v Can we hold off on this for now? Need more feedback.

**Comment by surbhigupta12** [comment]
@vikramjit-singh-chug, 
A gentle reminder.

**Comment by Rajeshwari-v** [comment]
> @Rajeshwari-v Can we hold off on this for now? Need more feedback.

Sure @heath-hamilton.

**Comment by Rajeshwari-v** [comment]
@vikramjit-singh-chug, a gentle reminder to review and approve the PR.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md) | 90 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e1abc350-a13a-4b7b-aca5-0edd5c63afce) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/design/bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/bots/design/bots.md) | 91 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c7af3e95-95c2-4a7e-a3a4-730846ef39c1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/activity-feed-notifications.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/concepts/design/activity-feed-notifications.md) | 92 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3c9ae28f-15ca-461a-b9af-040ff57c0dcc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-teams-app-ui-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/concepts/design/design-teams-app-ui-templates.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d3ea5b02-b83a-419f-b74d-9c4adebd69ed) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/personal-apps.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/concepts/design/personal-apps.md) | 93 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b30855e6-9818-4f93-a995-783aba9b510c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/device-capabilities/native-device-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/concepts/device-capabilities/native-device-permissions.md) | 89 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f30a4b4b-c565-4534-9afc-fbf5d55feae3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/design/messaging-extension-design.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/messaging-extensions/design/messaging-extension-design.md) | 89 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e8051a5c-4791-438d-a649-2d3453d5fa2a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/design/tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/tabs/design/tabs.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/42f32294-97e7-4609-9fe2-d3e633fc3e3f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/cards/design-effective-cards.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/task-modules-and-cards/cards/design-effective-cards.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0fc0c0db-e6c6-4c12-89b7-9a6443dec122) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/task-modules/design-teams-task-modules.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e55a78ca2750c8c5a1c485c89ec0098fa2abd4e/msteams-platform/task-modules-and-cards/task-modules/design-teams-task-modules.md) | 88 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7abae6da-a1e8-47c3-b982-59462068e159) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by heath-hamilton** [comment]
Starting this work over in another branch. Closing the PR,

**Pull request closed without merging**